test/full.js

```diff
@@ -664,9 +664,7 @@ test('Uses dataType compare', function (t) {
 
     compareRun = false;
     var foo = new Foo({ silliness: 'you' });
-    t.assert(compareRun);
```

Comment:
Could probably more properly be `t.notOk(compareRun)` so we're enforcing this optimization in the tests.